core/domain/html_cleaner.py

```diff
@@ -25,6 +25,14 @@
 
 
 def filter_a(name, value):
+    """Returns whether the given attribute of an anchor ('a') tag should be 
+    whitelisted.
+    Args:
```

Comment:
Here and below: There should be a single newline above "Args", and a single newline above "Returns".